apps/base-docs/src/components/Hero/TreatmentHero/BarChart/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+export default function BarChart() {
+  const maxValue = Math.max(...chartData.map((item) => item.value));
+
+  return (
+    <div>
+      <div className={styles.chartColumnsContainer}>
+        {chartData.map((chartColumn, index) => {
+          return (
+            <ChartColumn
+              key={chartColumn.name}
+              columnData={chartColumn}
+              columnHeight={(chartColumn.value / maxValue) * 100}
+              displayType={
+                index === 0
+                  ? 'chartColumnBase'
+                  : index % 2 === 0
+                  ? 'chartColumnEven'
+                  : 'chartColumnOdd'
+              }
+            />
+          );
+        })}
+      </div>
+      <div className={styles.chartLegendContainer}>
+        <div className={styles.chartLegend}>
+        Cost per transaction based on 90 day moving average.
+        </div>
+        <a className={styles.chartLegend} href="https://l2beat.com/scaling/costs" target="_blank">
+          Source: L2Beat
+        </a>
+      </div>
+    </div>
+  );
+}
+
+function ChartColumn({ columnData, columnHeight, displayType }: ChartColumnProps) {
+  return (
+    <div className={styles.chartColumnContainer}>
+      <div className={styles.chartColumnValue}>${columnData.value}</div>
+      <div style={{ height: `${columnHeight}%` }} className={styles[displayType]}>
+        <div className={styles.chartColumnName}>{columnData.name.toUpperCase()}</div>
+      </div>
+    </div>
+  );
+}
+
+const chartData = [
```

Comment:
In the future it'd be cool to fetch this data. Assuming the experiment goes well these numbers will go out of date.

Reply:
Agreed. I've spent a bit of time trying to find a source, let's discuss.